### PR TITLE
BaSP TG-22: Shared Button Component

### PR DESCRIPTION
**Pull request by Baldizan**
Button implementation:
image

### CODE REVIEW

**Review by nailaghigo — APPROVED** [review]
good job!!

**Review by LucianaPrat — APPROVED** [review]

**Review by Martinzancocchia — APPROVED** [review]

File: src/Components/Shared/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './button.module.css';
+
+const Button = ({ type = 'button', label, onClick, disabled, style, hidden, theme, icon }) => {
```
Comment (LucianaPrat) [inline src/Components/Shared/Button/index.js:4]: It would be good if the default theme is primary

Comment (Baldizan) [reply src/Components/Shared/Button/index.js:4]: Thanks!

**Review by mr-valen — APPROVED** [review]
Wow nice style!

**Pull request merged**